webui/react/typings/plotly.d.ts

```diff
@@ -1,3 +1,3 @@
-declare module 'plotly.js-basic-dist' {
-  export { Data, Layout } from '@types/plotly.js';
+declare module 'plotly.js/lib/core' {
+  export * from '@types/plotly.js';
```

Comment:
this might be overadvertising some types that core doesn't have

Reply:
like you suggested, let's keep an eye out for this becoming a problem and we'll then switch over to explicitly defining everything we want to use. Really hoping we don't come to that because that can also get unwieldy! 🤞